packages/engine-client/src/EngineExecuteClient.ts

```diff
@@ -96,4 +96,13 @@ export class EngineExecuteClient extends EngineBaseClient {
       await this.payloadBuilder.buildLinkSignerPayload(params),
     );
   }
+
+  async transferQuote(
```

Comment:
Should we document (either here or at the top-level client) the restrictions for this? (ex must be to same address, etc.)